### PR TITLE
backport documentation link fix

### PR DESCRIPTION
**Pull request by 0joshuaolson1**

### CODE REVIEW

**Comment by MrMormon** [comment]
This has been corrected already in beta and nightly, but thank you bot.

**Comment by alexcrichton** [comment]
Thanks for the PR! Currently we don't issue new point releases (aka update documentation as well) for minor fixes like this, but this should be fixed once beta makes its way to stable soon!

**Comment by MrMormon** [comment]
@alexcrichton Are there any plans to fix such an obvious bug in the six-week release process?

**Comment by steveklabnik** [comment]
Six weeks is significantly faster than most languages release; if this was Ruby, this bug would stick around for a year.

We don't generally do backports unless they're _really_ important. We've never issued a `z` version release yet. Stable is _stable_, but that doesn't mean it's perfect. Nothing is.

**Comment by MrMormon** [comment]
Feel free to spend your time more productively than a debate, but is the philosophy of _stable_ more important than fixing the online docs at least?

**Comment by steveklabnik** [comment]
>  is the philosophy of stable more important than fixing the online docs at least?

In general, yes. All software will have bugs. It's unavoidable. But this is one of the reasons we do the six-week cycle in the first place: we'd like new changes to have the six weeks of beta time, to make sure that things shake out. Any hotfixes we make haven't gone through that process, and so they could go wrong.

Now, for a small docs fix like this? Very unlikely to break something. But once you decide to have a policy of "some non-security fixes will get backported",  you have to also figure out which ones should and which ones shouldn't. So yeah, we could say "small" doc fixes get backported, but then which ones are truly small? And then how do you know when to release a new `z` version, after every one? And this also means that rather than regression fixes being applied to beta, you now have tons of more, which is more work, both to actually do the backport, check it, and merge in, but also now backporting is something that needs to be checked regularly.

Basically, it messes up the whole idea that beta is a release candidate and complicates the workflow, just to fix minor issues (they have to be minor, that's why they'd be eligible for a backport in the first place). This doesn't seem like a good tradeoff, especially when you're going to see the fix at most 12 weeks from now, which is already pretty quick.

Does that make mor esense?

**Comment by MrMormon** [comment]
Knowing that Rust has a documented policy on this and I won't convince anyone otherwise is good enough for me. Thanks for getting back.

**Comment by aturon** [comment]
@steveklabnik I wonder if it'd be worth taking your post and creating an entry on the [Forge](https://forge.rust-lang.org/) related to it? Perhaps it could be part of a growing policies page.

**Comment by MrMormon** [comment]
@aturon I didn't know about the Forge. Does the main website link to it?

**Comment by aturon** [comment]
Not yet -- it's still a pretty early work in progress. But once it fills
out a bit more, I expect we'll link more prominently to it.

**Comment by MrMormon** [comment]
This is my favorite: https://forge.rust-lang.org/bibliography.html
